nncf/experimental/onnx/graph/model_transformer.py

```diff
@@ -122,7 +122,8 @@ def _get_extra_model_outputs(self,
                 if transformation.target_point.type == TargetType.POST_LAYER_OPERATION:
                     edge_name = onnx_graph.get_node_edge_names(node_name)['output'][0]
                 elif transformation.target_point.type == TargetType.PRE_LAYER_OPERATION:
-                    edge_name = transformation.target_point.edge_name
+                    edge_name = onnx_graph.get_node_edge_names(node_name)['input'][
```

Comment:
Still `edge_name`?

Reply:
Here we need `edge_name` for registering new outputs for ten model

Reply:
But why everywhere in the other files it was changed into `port_id` and here is not?

Reply:
Because at this part of the code we get name of the edge, not port number